csm_web/scheduler/models.py

```diff
@@ -173,29 +170,22 @@ class Meta:
 
 
 class Spacetime(ValidatingModel):
-    MONDAY = "Mon"
-    TUESDAY = "Tue"
-    WEDNESDAY = "Wed"
-    THURSDAY = "Thu"
-    FRIDAY = "Fri"
-    SATURDAY = "Sat"
-    SUNDAY = "Sun"
-    DAY_OF_WEEK_CHOICES = (
-        (MONDAY, "Monday"),
-        (TUESDAY, "Tuesday"),
-        (WEDNESDAY, "Wednesday"),
-        (THURSDAY, "Thursday"),
-        (FRIDAY, "Friday"),
-        (SATURDAY, "Saturday"),
-        (SUNDAY, "Sunday"),
-    )
-    DAY_INDEX = tuple(day for day, _ in DAY_OF_WEEK_CHOICES)
+    class DayOfWeek(models.TextChoices):
```

Comment:
Specifying the labels here is unnecessary, if omitted Django will assume it should be the capitalized version of the enum name. So `MONDAY = "Mon"` would suffice, for example